From 1e42d3de4f554e69850cc9c10ab564813484fbff Mon Sep 17 00:00:00 2001
From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Thu, 15 Jul 2021 16:12:23 +0200
Subject: [PATCH] Makefile: add variable for config directory

For some distros (including Debian), the Firefox config directory can be
different from `/etc/firefox`. Adding a variable for this location so it can
be easily customized at build time.

---
 Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9e22b91..03d5d30 100644
--- a/Makefile
+++ b/Makefile
@@ -8,6 +8,7 @@ HOMEPAGE_FILES := head.html distro_links.html bottom.html
 DISTRO := postmarketOS
 DESTDIR :=
 FIREFOX_DIR := /usr/lib/firefox
+FIREFOX_CONFIG_DIR := /etc/firefox
 
 all: out/home.html out/userChrome.css out/userContent.css
 
@@ -29,7 +30,7 @@ out/userContent.css: $(USERCONTENT_FILES) out
 
 install: all
 	install -Dm644 src/policies.json \
-		"$(DESTDIR)/etc/firefox/policies/policies.json"
+		"$(DESTDIR)/$(FIREFOX_CONFIG_DIR)/policies/policies.json"
 	install -Dm644 src/mobile-config-prefs.js \
 		"$(DESTDIR)/$(FIREFOX_DIR)/defaults/pref/mobile-config-prefs.js"
 	install -Dm644 src/mobile-config-autoconfig.js \
-- 
2.30.2
